stories/Theme/Theme.story.mdx

```diff
@@ -0,0 +1,142 @@
+import { Meta } from '@storybook/addon-docs';
+
+import { Header } from '../shared'
+
+import { BreakpointsProperties, ColorIntensity, SpacingValues } from './Theme';
+
+<Meta 
+  title="Foundation/Theme" 
+  parameters={{
+    previewTabs: { 
+      'storybook/docs/panel': null, 
+      canvas: {
+        hidden: true,
+      },
+    },
+  }}
+/>
+
+<Header title="Theme">
+  All Quantum components are initialized with the default Quantum theme.
+  <br />
+  Here we will describe which properties are available on it.
+</Header>
+
+## Quantum Theme
+
+The theme has properties which, some of them, are used in components.
+It's structured with these ones:
+
+- [breakpoints](#breakpoints)
+- [colors](#colors)
+- [gutter](#gutter)
+- [baseFontSize](#baseFontSize)
```

Comment:
I suggest checking this reference

Reply:
Good point, but in this case you can request change because this causes a bug.

Reply:
Right!